optimum/habana/transformers/models/t5/modeling_t5.py

```diff
@@ -516,6 +516,7 @@ def gaudi_T5ForConditionalGeneration_forward(
     return_dict: Optional[bool] = None,
     cache_position: Optional[torch.LongTensor] = None,
     token_idx: Optional[torch.LongTensor] = None,
+    num_items_in_batch:Optional[int] = None,
```

Comment:
```suggestion
    **kwargs,
```

@libinta For several models, we are utilizing `**kwargs` to handle variable keyword arguments. I recommend continuing this approach instead of using specific parameters like `num_items_in_batch` to maintain consistency and flexibility across our codebase.